WooCommerce/src/main/kotlin/com/woocommerce/android/util/StringUtils.kt

```diff
@@ -103,6 +105,16 @@ object StringUtils {
         } ?: false
     }
 
+    /**
+     * Returns true if the passed string is an Automattic/A8c email
+     */
+    fun isA8cEmail(email: String?): Boolean {
+        if (email != null && (AUTOMATTIC in email.lowercase() || A8C in email.lowercase())) {
+            return true
+        }
+        return false
```

Comment:
Maybe we could reduce it to 
```suggestion
        return email != null && (AUTOMATTIC in email.lowercase() || A8C in email.lowercase())
```
?